tests/storage/datavolume.go

```diff
@@ -381,6 +382,92 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 					return vm.Status.PrintableStatus
 				}, 180*time.Second, 2*time.Second).Should(Equal(v1.VirtualMachineStatusStopped))
 			})
+
+			It("should accurately aggregate DataVolume conditions from many DVs", func() {
+				sc, err := libstorage.GetSnapshotStorageClass(virtClient)
+				if err != nil {
+					Skip("no snapshot storage class configured")
+				}
+
+				dataVolume1 := libdv.NewDataVolume(
+					libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine)),
+					libdv.WithPVC(libdv.PVCWithStorageClass(sc)),
+				)
+				dataVolume2 := libdv.NewDataVolume(
+					libdv.WithRegistryURLSource(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine)),
+					libdv.WithPVC(libdv.PVCWithStorageClass(sc)),
+				)
+
+				vmi := tests.NewRandomVMIWithDataVolume(dataVolume1.Name)
+
+				By("requiring an additional DataVolume")
+				diskName := "disk1"
+				vmi = libstorage.AddDataVolumeDisk(vmi, diskName, dataVolume2.Name)
+
+				vmSpec := tests.NewRandomVirtualMachine(vmi, true)
+
+				vm, err := virtClient.VirtualMachine(vmSpec.Namespace).Create(context.Background(), vmSpec)
+				Expect(err).ToNot(HaveOccurred())
+
+				Eventually(func() v1.VirtualMachinePrintableStatus {
```

Comment:
There is a matcher for the printable status in `vm_test.go`. Maybe you could use `ThisVM` and the `printableStatus` matcher here too?

Reply:
done